WBooks/LibraryView.swift

```diff
@@ -12,6 +12,7 @@ import WolmoCore
 
 class LibraryView: UIView, NibLoadable {
     
+    @IBOutlet weak var tblBooks: UITableView!
```

Comment:
Change `tblBooks` to `tableBooks`. It's always better to write complete names.

Reply:
Done